ls1intum/Artemis
# `Quiz exercises`: Show error message for too long answer options in short answer questions

### Checklist
#### General


#### Server
#### Client
#### Changes affecting Programming Exercises

The unticked steps are not relevant for this PR.


### Motivation and Context


If fixes the Issue #4177 
The problem was that a short answer question does not accept >= 250 characters (for reference this is the same is the for the title), but no error message was shown.

### Description

I added a new error message which gets shown in the bottom in the `Invalid Input` field. I translated into in English and German. I added a corresponding test as well.

### Steps for Testing

Prerequisites:
- 1 Instructor

1. Log in to Artemis
2. Navigate to Course Administration
3. Navigate to Exercises
4. Create a new Quiz Exercise
5. Add a `Short Answer Question`
6. Verify that the new error message is not shown if the length for all options is less than 250 characters
7. Enter >= 250 characters for at least one answer option 
8. Verify that the new error message is shown

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage






### Screenshots



<img width="1440" alt="Error Message" src="https://user-images.githubusercontent.com/82283581/138685549-5e2fc7ac-1a59-46f4-8452-bb5d608f027e.png">
